packages/spotlight/SpotlightRootDecorator/SpotlightRootDecorator.js

```diff
@@ -53,14 +64,20 @@ const defaultConfig = {
  * @hoc
  */
 const SpotlightRootDecorator = hoc(defaultConfig, (config, Wrapped) => {
-	const {noAutoFocus} = config;
+	const {getInputTypeSetter, noAutoFocus, rootId} = config;
+	const rootNode = typeof document === 'object' && document.querySelector('#' + rootId) || document;
+	let lastInputType = 'key';
+	let inputTypeChanged = false;
+	let inputTypeActivated = false;
```

Comment:
I think we'd better have these in `inputType` object.

Reply:
🆗